addons/binding/org.openhab.binding.tadoac/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is generated by the archetype and afaik, we never have a license header on it in any other binding...